### PR TITLE
Add news post for playtest-20170930.

### PR DESCRIPTION
**Pull request by pchote**

### CODE REVIEW

File: content/news/playtest-20170930.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+* Fixed in-game map installation on Linux and Windows
+* Fixed a crash when using Attack Move
+* Fixed incorrect global chat timestamps when returning to the lobby after a match
+* Added RA minelayer deployment to the command-bar
+* Improved attack-move cursor behaviour outside the map
+
+See the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/) if you are interested in finding more details on these fixes, or head on over to our [download page](http://www.openra.net/download/) to get the new release!
+
+This playtest will be your last chance to report any bugs or balance issues, so we encourage everybody to play a few games, and let us know of any issues via [our forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), our [GitHub issue tracker](https://github.com/OpenRA/OpenRA/issues), or in the comments below.
```
Comment (Phrohdoh) [inline content/news/playtest-20170930.md]: `This playtest will be your last chance to report any bugs or balance issues` should, IMO, end with `before a stable release is made available`

File: content/news/playtest-20170930.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Thanks to everybody who reported issues with [playtest-20170923](http://www.openra.net/news/playtest-20170923/) last week, today we are releasing another playtest with several more fixes.  [Playtest 20170930](http://www.openra.net/download/) includes:
+
```
Comment (reaperrr) [inline content/news/playtest-20170930.md]: Is the 2nd space before the 2nd sentence intentional?

**Comment by pchote** [comment]
Updated again.

**Comment by pchote** [comment]
Updated with release ID and frozen changelog link.

**Comment by Mailaender** [comment]
Updated https://build.opensuse.org/package/show/games:openra/playtest

**Pull request merged**